docs/source/installer/packstack.rst

```diff
@@ -0,0 +1,44 @@
+Packstack
+---------
+
+Infrared allows to use the Packstack to install Openstack::
+
+    $ ir-installer -d -vvvv --inventory hosts packstack --product-version=8 -o install.yml -e @provision.yml -e @private.yml
+
+Here required arguments are:
+    --product-version: the product version to install.
+
+Optional arguments:
+    * provision.yml - the settings file generated ny provisiner. I might contains required for Packstack settings.
+    * private.yml - the settings file with the private properties.
+
+
+Settings structure
+^^^^^^^^^^^^^^^^^^
+
+The main settings file for packstack installer is located in the::
```

Comment:
The path for the main settings file for packstack installer::

  settings/....